### PR TITLE
update pub/sub example and fix

### PR DESCRIPTION
**Pull request by semigodking**
For the dirty fix, hopefully you have better idea. Thanks.

### CODE REVIEW

File: examples/publish_subscribe_example.py [hunk 1]
```diff
@@ -16,8 +16,7 @@ def clock(rpc):
     """
 
     while True:
-        rpc.notify('clock', str(datetime.datetime.now()))
-
+        yield from rpc.notify('clock', str(datetime.datetime.now()))
```
Comment (fscherf) [inline examples/publish_subscribe_example.py]: Could you replace all `yield from` with `await` when you touch this file anyway? :)

File: examples/publish_subscribe_client.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+from aiohttp_json_rpc.client import JsonRpcClient
+
+
+async def handler(data):
+    if data['method'] == 'clock':
+        print(data)
+
+
+async def run():
+    message = asyncio.Future()
```
Comment (fscherf) [inline examples/publish_subscribe_client.py]: This line is unnecessary. Please run the linter before pr
``tox -e lint``

**Comment by semigodking** [comment]
That line is the only output.  No exception traceback.

**Comment by fscherf** [comment]
Did you get that output while running a example or custom code?

**Comment by fscherf** [comment]
ah! i can reproduce your problem

**Comment by fscherf** [comment]
It seems you discoverd a design issue here: if a websocket client does no graceful shutdown but disconnects hard, the rpc does not recognize it proberly. the client remains in the list of connected clients

**Comment by fscherf** [comment]
@semigodking 

I pushed a branch with a modified version of your patches (`topic/non-graceful-disconnect`).
If you are ok with my changes, i'll merge it like that

**Comment by semigodking** [comment]
Hi @fscherf
<https://github.com/pengutronix/aiohttp-json-rpc/commits?author=fscherf>,
that looks good to me. Thank you!

**Pull request closed without merging**